crates/hyperswitch_connectors/src/connectors/fiuu/transformers.rs

```diff
@@ -991,6 +991,7 @@ pub struct FiuuRefundSuccessResponse {
     #[serde(rename = "RefundID")]
     refund_id: i64,
     status: String,
+    reason: Option<String>,
```

Comment:
Are we getting this reason in response. Did you test it?

Reply:
no @deepanshu-iiitu , not able to test this . `reason`  is an optional field and will only come in unsucessful refund cases . Not able to test  thisunsuccesful refund flow locally

Reply:
You could reach out to @awasthi21  to check if he knows how to trigger unsuccessful refunds.